tests/integration/components/frost-select-test.js

```diff
@@ -92,78 +93,67 @@ describeComponent(
     })
 
     it('lists all passed data records', function () {
-      run.later(() => {
-        expect(this.$('.frost-select li').length).to.eql(props.data.length)
-      })
+      expect(this.$('.frost-select li').length).to.eql(props.data.length)
```

Comment:
This file has a lot of these, basically I just swapped code that looked like this:

```js
// do something
run.later(() => {
  // check something
})
```
To look like this: 

```js
run(() => {
  // do something
})
// check something
```

Making sure all the `// do something` happen in a run-loop instead of waiting till the next run loop to `// check something`